migrations/1457249988959-add-roles.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// load .env
+require('dotenv').load();
+
+const Roles = require('../server/models/roles');
+const titles = Roles.schema.paths.title.enumValues;
+
+exports.up = function(next) {
+
+  let tasks = titles.map((title) => {
+    let query = {title: title};
+    let update = {
+      title: title
+      //accessLevel: 0 // FIXME: Is there a default accessLevel?
```

Comment:
*Clarification*

The access levels should be as follows:

- user: 0
- staff: 1
- admin: 2

I'm not sure how we could specify defaults for this 
@aksalj any ideas?